logback-contrib/pom.xml

```diff
@@ -0,0 +1,366 @@
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.linagora.tmail</groupId>
+        <artifactId>tmail-backend-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <groupId>com.linagora.tmail</groupId>
+    <artifactId>logback-contrib-parent</artifactId>
+    <packaging>pom</packaging>
+
+    <name>Logback Contrib</name>
+    <url>https://github.com/qos-ch/logback-contrib/wiki</url>
+    <description>
+        Logback Contrib provides community-driven and supported
+        extensions to the Logback logging framework. All contributions
+        are made by contributors with CLA on file.
+    </description>
+    <inceptionYear>2012</inceptionYear>
+
+    <!--
+   <licenses>
+     <license>
+       <name>Eclipse Public License - v 1.0</name>
+       <url>http://www.eclipse.org/legal/epl-v10.html</url>
+     </license>
+
+     <license>
+       <name>GNU Lesser General Public License</name>
+       <url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html</url>
+     </license>
+   </licenses>
+   -->
+
+    <scm>
+      <connection>scm:git:git://github.com/qos-ch/logback-contrib.git</connection>
+      <developerConnection>scm:git:git@github.com:qos-ch/logback-contrib.git</developerConnection>
+      <url>https://github.com/qos-ch/logback-contrib</url>
+      <tag>HEAD</tag>
+    </scm>
+    <issueManagement>
+      <system>GitHub</system>
+      <url>https://github.com/qos-ch/logback-contrib/issues</url>
+    </issueManagement>
+
+    <developers>
+        <developer>
+            <id>lhazlewood</id>
+            <name>Les Hazlewood</name>
+            <email>les@hazlewood.com</email>
+            <url>http://www.leshazlewood.com</url>
+            <timezone>-8</timezone>
+        </developer>
+        <developer>
+            <id>belaso</id>
+            <name>Christian Trutz</name>
+            <email>christian.trutz@belaso.de</email>
+            <url>http://www.belaso.de</url>
+            <timezone>+2</timezone>
+        </developer>
+        <developer>
+            <id>tony19</id>
+            <name>Tony Trinh</name>
+            <email>tony19@gmail.com</email>
+            <url>http://tony19.github.com</url>
+            <timezone>-5</timezone>
+        </developer>
+    </developers>
+    <contributors>
+    	<contributor>
+    		<name>David J. M. Karlsen</name>
+    		<email>david@davidkarlsen.com</email>
+    	</contributor>
+        <contributor>
+            <name>Espen A. Fossen</name>
+            <email>espenaf@junta.no</email>
+        </contributor>
+    </contributors>
+
+    <modules>
+        <module>json</module>
+        <module>jackson</module>
+    </modules>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <!-- Compile Dependencies: -->
+        <logback.version>1.4.14</logback.version>
+        <jackson.version>2.17.1</jackson.version>
```

Comment:
Logback and jackson version shall be inherited from parent POM
If undefined in parent pom then we shall rely on APache James "Bill Of Materials"

How to use a bill of materials: https://github.com/linagora/tmail-backend/blob/2d5a869a748390afe923b22a52bd69912e2c2574/tmail-backend/pom.xml#L327

Reply:
> Logback and jackson version shall be inherited from parent POM

Not done!